mod/messageboard/start.php

```diff
@@ -114,7 +114,11 @@ function messageboard_add($poster, $owner, $message, $access_id = ACCESS_PUBLIC)
 			$poster->getURL()
 		), $owner->language);
 
-		notify_user($owner->guid, $poster->guid, $subject, $body);
+		$params = [
+			'action' => 'create',
```

Comment:
I don't see any issues using full action name. Translation strings and hook names are strings, so having `notification:messageboard/add' is not an issue.
I don't like the magic translation strings as you have no control over what parameters are passed to sprintf.